R/calc.lw.net.R

```diff
@@ -127,7 +127,7 @@ calc.lw.net.base <- function(dateTime,sw,Ts,lat,atm.press,airT,RH){
   cl2 <- 1.22+0.06*sin((month+2)*pi/6)
   cl3 <- vp/T_k
   T_k1 <- T_k^4
-  S_B <- 5.67E-8 # Stefan-Boltzman constant (°K is used)
```

Comment:
Yeah, CRAN now checks for non-ascii characters in *.R files, which it didn't do before.